source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -81,14 +79,29 @@ namespace aspect
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value.  Units: None");
-
         prm.declare_entry ("Pressure prefactors for Frank Kamenetskii", "0.0",
                            Patterns::List(Patterns::Double (0.)),
                            "A prefactor for the pressure term in the viscosity approximation, "
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value. "
                            "Units: None");
+        prm.declare_entry ("Reference Temperature for Frank Kamenetskii", "0.0",
```

Comment:
Please change the name of the variable to be consistent with our usual naming convention (capital starting letter, all small letters otherwise except for proper nouns): `Reference temperatures for Frank Kamenetskii`. Also 0.0 is a dangerous default, because it could be used in non-dimensional models. You can do:
```suggestion
        prm.declare_entry ("Reference temperatures for Frank Kamenetskii", boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
```

And then below in `parse_parameters` you can then compare the value to `std::numeric_limits<double>::max()`.